dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -65,7 +65,11 @@
     {
         "TUNNEL_DECAP_TABLE:IPINIP_SUBNET" : {
             "tunnel_type":"IPINIP",
+{% if "mlnx" in DEVICE_METADATA.localhost.platform or "cisco" in DEVICE_METADATA.localhost.platform %}
```

Comment:
do we need to add arista or bcm here? @abhishekd-brcm

Reply:
Please don't add mlnx for dscp_mode. Mlnx devices support pipe mode.

Reply:
We should then have a check excluding brcm. Do we know if other vendors too have the same restriction?